server/src/modules/users/domain/entities/auth-code/auth-code.ts

```diff
@@ -0,0 +1,38 @@
+import { Result } from '../../../../../shared/core/result'
+import { AggregateRoot } from '../../../../../shared/domain/aggregate-root'
+import { UniqueEntityID } from '../../../../../shared/domain/unique-entity-id'
+import { AuthCodeCreated } from '../../events/auth-code-created'
+import { AuthCodeString } from '../../value-objects/auth-code'
+
+interface AuthCodeProps {
+  clientId: string,
+  authCode: AuthCodeString
```

Comment:
Property in question of https://github.com/loolabs/auth/pull/2#discussion_r649664338